sw/device/lib/testing/test_framework/freertos_port.S

```diff
@@ -15,6 +16,40 @@
 
 // -----------------------------------------------------------------------------
 
+/**
+ * Save MEPC to the stack.
+ *
+ * NOTE: this IRQ is synchronous, therefore, we must update the ISR return
+ * address to point to the instruction after the one that triggered this IRQ.
+ * Since we support the RISC-V compressed instructions extension, we need to
+ * check if the two least significant bits of the instruction are
+ * b11 (0x3), which means that the trapped instruction is not compressed,
+ * i.e., the trapped instruction is 32bits = 4bytes. Otherwise, the trapped
+ * instruction is 16bits = 2bytes.
+ */
+
+.balign 4
+.type _process_mepc_on_synchronous_irq, @function
+_process_mepc_on_synchronous_irq:
```

Comment:
Is there a reason for the leading underscore? This has internal linkage after all.

Reply:
It was a way to signify it was a private sub-routine (wasn't sure if this was convention or not).

Removed.